### PR TITLE
Adds a warning if trying to ignite remove /path/to/plugin

### PR DESCRIPTION
**Pull request by jamonholmgren**
Per #702 , this PR adds a warning if you try to remove via a path.

screen shot 2017-02-26 at 7 02 09 pm

I also fixed a bug where doing `ignite remove ignite-<plugin>` would try to remove `ignite-ignite-<plugin>`, and also refactored the prepending function to be used consistently from everywhere. Also a bit less Rambda/Rambdasauce and more vanilla JS.

I then snuck in a launch deadline update to the README. Steve will kill me for so many unrelated things in this PR if he notices.

### CODE REVIEW

File: packages/ignite-cli/src/commands/remove.js [hunk 1]
```diff
@@ -3,29 +3,30 @@
...
 
 const existsLocally = (moduleName) => {
   // we take a look at the local package.json
   const pack = require(`${process.cwd()}/package.json`)
-  return R.pathOr(null, ['devDependencies', moduleName], pack)
+  return pathOr(null, ['devDependencies', moduleName], pack)
 }
 
-const noMegusta = (moduleName) => {
+const removeDependency = (moduleName) => {
```
Comment (jamonholmgren) [inline packages/ignite-cli/src/commands/remove.js]: I know, I'm no fun.

File: packages/ignite-cli/src/lib/detectInstall.js [hunk 2]
```diff
@@ -1,18 +1,4 @@
...
- *
- * @param {string} value The string to check.
- * @returns {string} The same string, but better.
- */
-const alwaysStartWithIgnite = function (value) {
-  return unless(
-    startsWith('ignite-'),
-    concat('ignite-')
-  )(value)
-}
```
Comment (jamonholmgren) [inline packages/ignite-cli/src/lib/detectInstall.js]: Turns out a simple regex & ternary works just as well.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 All active development is taking place on the `master` branch.  If you're contributing to v2, welcome!  You're in the right place.
 
-We're aiming to release around the end of February.
+We're aiming to release around the first week of March.
```
Comment (skellock) [inline README.md]: This will work to our advantage.  Once 42 comes out.

File: README.md [hunk 4]
```diff
@@ -15,15 +15,13 @@ We're aiming to release around the end of February.
 <p align="center">
   :fire: The ideal starting app for React Native, best practices, generators, and more. :fire:
   <br/>
-  <a href="https://infiniteredcommunity.herokuapp.com/">
+  <a href="https://community.infinite.red/">
```
Comment (skellock) [inline README.md]: The secure site doesn't seem to work.  `http` does tho.

Comment (jamonholmgren) [inline README.md]: Fixed on `master` directly.

**Pull request merged**